content/04.results.md

```diff
@@ -186,3 +186,30 @@ Next the app preprocesses the requested preprint and returns a listing of (**B**
 **D.** The app also displays the location of the query preprint in PMC.
 **E.** Users can select a square within the landscape to examine statistics associated with the square including the top journals by article count in that square and the odds ratio of tokens.
 ](images/journal_recommender_workflow.png){#fig:journal_rec_workflow width="100%"}
+
+### Contexualizing Preprints in Motion
+
+![
+**A.** Published articles are focused on COVID-19 nomenclature and epidemiology while preprints are focused on experimental setup and instruments.
+We plotted the 95% confidence interval for each reported token.
+**B.** "fviii" and "collagen" are more prevalent within preprints while "s." "$\pm$" appear more frequently within published articles.
+**C.** Preprints are largely clustered within the center of the published landscape.
+This square bin plot depicts the binning of all published papers within the PMCOA corpus.
+High-density regions are depicted in yellow while low-density regions are in dark blue.
+Red dots represent bioRxiv preprints that have been analyzed in [@doi:10.1101/2021.02.20.432090].
+**D.** Preprints analyzed in [@doi:10.1101/2021.02.20.432090] are published faster than regular bioRxiv preprints.
+The x-axis shows the number of versions of a preprint that were posted on bioRxiv and the y-axis shows the number of days that elapsed between when the first version of a preprint was posted on bioRxiv and the date at which the peer-reviewed publication appeared.
+The density of observations is depicted in the violin plot with an embedded boxplot.
+The red dots represent preprints analyzed in [@doi:10.1101/2021.02.20.432090].
+**D.** Majority of preprints analyzed in [@doi:10.1101/2021.02.20.432090] are published faster than regular bioRxiv preprints.
+The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form.
+The y-axis shows the number of days elapsed between when the first version of a preprint is posted on bioRxiv and the time a preprint is published.
+The color bar on the right represents the density of each hexbin in this plot where more dense regions are shown in a brighter color.
+The red dots represent preprints analyzed in [@doi:10.1101/2021.02.20.432090].
+]
+(https://raw.githubusercontent.com/danich1/annorxiver/add_last_figure/figure_generation/output/figure_five_panels.svg){#fig:covid_19_preprints width="100%"}
+
+Preprints posted from January 1st, 2020 to April 30th, 2020 timeframe underwent moderately impacted during the peer review process and were published within a timely manner.
```

Comment:
This results section doesn't really provide much context. Here are things that I see in the figures:
* It looks like the terminology change from 2019-ncov to sars-cov-2 is captured in the first figure, which would have happened as these articles were being published. That's an interesting observation.
* Also typesetting with +/- and emdash still shows up
* C isn't showing up for me but I'm guessing that's because the landscape is being updated.
* For D can you fit a trendline for the red points and show it in red? I am guessing there is not a relationship between number of versions and time to publish, which would be a difference from what you reported before and that seems interesting.
* E looks like it _might_ still have a difference between linguistic change and time to publish (e.g., a significant slope), but it'd be grand to do the same quantitative analysis there.

Is there anything else that you see in the figures that should be commented on?